### PR TITLE
Support attributed strings in picker rows

### PR DESCRIPTION
**Pull request by anlaital**
This adds support for specifying titles for picker rows as attributed strings, allowing for further customisation of how the picker displays its titles. If `onProvideStringAttributes ` is not set, then previous functionality is retained.

### CODE REVIEW

File: Source/Rows/PickerRow.swift [hunk 1]
```diff
@@ -75,7 +75,7 @@ open class PickerCell<T> : Cell<T>, CellType, UIPickerViewDataSource, UIPickerVi
     open func numberOfComponents(in pickerView: UIPickerView) -> Int {
         return 1
     }
-
+
```
Comment (mats-claassen) [inline Source/Rows/PickerRow.swift]: Please remove this tab

Comment (anlaital) [reply Source/Rows/PickerRow.swift]: That should be a newline, or am I missing something? 😅

**Comment by anlaital** [comment]
This should now be fixed.

**Comment by mtnbarreto** [comment]
Closing due to lack of response.

**Pull request closed without merging**